### DISCUSSION TITLE
Adding Apple Privacy Manifest Support

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#15136] [General]
Ref: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
Ref: https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/

**Comment by maxkatz6** [comment]
Ideally, default .NET manifest will work. And only platform APIs like storage provider would require more.

Long term, we should add this manifest to our templates. When we will know what exactly should be written there.
